### PR TITLE
Polish

### PR DESCRIPTION
**Pull request by naterush**
1. Adds a README, with instructions for setup
2. Adds setup.bat and run.bat scripts. I need to test these on windows, which I will do in my great windows testing tomorrow (but feel free to test them tomorrow)
3. Saves the automations in a `automations` folder, under a safe file path, with a bunch more metadata. 
4. Improves the robustness of the app across the board.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# MyPlanAdvocate Provider Recon App
+
+Need to check a new book of business against our salesforce data? This app makes it easy
+
+
+## Setting up Mito
+
+1. Make sure you have Python 3.8-3.11 installed, installing [here](https://www.python.org/downloads/release/python-3116/)
+2. Open a Command Prompt. Use the command `cd chnage\to\path\to\this\folder` to navigate to this folder.
```
Comment (aarondr77) [inline README.md]: chnge -> change

File: README.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+3. Run the command `setup.bat` -- this will either print "Setup completed successfully." or inform you of a setup error.
+
+## Running this app
+
+Now that everything is installed, run this app by:
+1. Open a Command Prompt. Use the command `cd chnage\to\path\to\this\folder` to navigate to this folder.
```
Comment (aarondr77) [inline README.md]: chnge -> change

File: setup.bat [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    echo Error: Python is not installed. Please install Python.
+    exit /b 1
+)
+
+REM Get Python version
+for /f "delims=" %%i in ('python -c "import platform; print(platform.python_version())"') do set PYTHON_VERSION=%%i
+
+REM Check Python version
+echo Found Python version %PYTHON_VERSION%
+if "%PYTHON_VERSION%" lss "3.8" (
```
Comment (aarondr77) [inline setup.bat]: I just tested this with Python version 3.11.6 and it printed the error message:

Found Python version 3.11.6
Error: Python version is too old. Please install Python between 3.8 and 3.11

Comment (naterush) [reply setup.bat]: @aarondr77 wanna just remove this check? You can do it manually -- it's kinda overkill.

**Pull request merged**